addOns/pscanrulesAlpha/src/main/javahelp/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html

```diff
@@ -37,6 +37,32 @@ <H2>Content Cacheability</H2>
 <p>
 Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/master/addOns/pscanrulesAlpha/src/main/java/org/zaproxy/zap/extension/pscanrulesAlpha/CacheableScanRule.java">CacheableScanRule.java</a>
 
+<H2>Cross-Origin-Resource-Policy Scan Rule</H2>
+Spectre is a side-channel attack allowing an attacker to read data
+from memory. One of the counter-measure is to prevent sensitive data
+to enter the memory.
+<blockquote cite="https://resourcepolicy.fyi/">
+ The Cross-Origin-Resource-Policy (CORP) header allows you
+ to control the set of origins that are empowered to include
+ a resource. It is a robust defense against attacks like Spectre,
+ as it allows browsers to block a given response before it enters
+ an attacker's process.
+</blockquote>
+For example, an attacker site can include a image tag with an attribute
+src to an internal content. The browser will load the data. With a
+side-channel attack, an attacker will be able to read it.
+
+CORP is complementary of CORB and COOP.
```

Comment:
```suggestion
CORP is complementary to CORB and COOP.
```

The later two should be spelled out and potentially linked.